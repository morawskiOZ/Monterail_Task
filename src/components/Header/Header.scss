@import "../../style/variables";
$header_height: 20px;

.Header {
  background-color: $main_blue;
  width: 100%;
  height: 115px;
  grid-template-rows: auto 1fr;
  &-Stripe {
    height: $header_height;
    background-color: $secondary_blue;
    width: 100%;
  }
  &-Title {
    height: calc(100% - #{$header_height});
    color: $white_opacity;
    display: grid;
    grid-template-columns: 1fr minmax(auto, 800px) 1fr;
    &--text {
      grid-column: 2;
      align-self: center;
      font-size: 36px;
    }
  }
}
