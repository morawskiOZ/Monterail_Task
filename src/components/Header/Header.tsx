import React from "react"
import "./Header.scss"

const Header = () => {










  
  return (
    <header className="Header App-content--entireRow">
      <div className="Header-Stripe" />
      <div className="Header-Title">
        <span className="Header-Title--text"> New event</span>
      </div>
    </header>
  )
}

export default Header
