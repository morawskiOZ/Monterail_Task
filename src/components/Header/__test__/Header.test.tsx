import { mount } from "enzyme"
import toJSON from "enzyme-to-json"
import React from "react"
import Header from "../Header"

describe("<Header/>", () => {
  it("renders and matches snapshots", async () => {
    const wrapper = mount(<Header />)
    expect(toJSON(wrapper.find("header"))).toMatchSnapshot()
  })
})
