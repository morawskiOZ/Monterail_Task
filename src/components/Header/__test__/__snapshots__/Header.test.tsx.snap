// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders and matches snapshots 1`] = `
<header
  className="Header App-content--entireRow"
>
  <div
    className="Header-Stripe"
  />
  <div
    className="Header-Title"
  >
    <span
      className="Header-Title--text"
    >
       New event
    </span>
  </div>
</header>
`;
